code/modules/awaymissions/pamphlet.dm

```diff
@@ -46,3 +46,31 @@
 			As a participant in the Nanotrasen Gateway Project, you will be on the frontiers of space. \
 			Though complete safety is assured, participants are advised to prepare for inhospitable \
 			environs."
+
+/obj/item/paper/pamphlet/cybernetics
+	name = "pamphlet - \'Synthman's Cybernetic Starter Gear!\'"
+	default_raw_text = "Join the Body Modder Revolution today! We are offering FREE SAMPLES of the latest and greatest \
+		cybernetic augments by Synthman Co. to you in this rare exclusive offer! With this letter, you are being gifted a \
+		special limited edition choice NTSDA-certified grade-A cybernetic implant, FREE OF CHARGE! Build up your body to \
+		GREATNESS with Synthman's new exclusive line of cybernetic products! Become greater, stronger, and BETTER today!"
+	var/obj/item/organ/internal/heart/cybernetic/sample
```

Comment:
Are you sure you do not need to set QDEL_NULL this? I know you have Exited setup but I'm not sure if that's called when the paper itself qdels.